packages/react/src/hooks/dashboard/useDispatchIntent.ts

```diff
@@ -64,7 +64,7 @@ type UseDispatchIntentParams =
  *   - `intentId` - Specific ID of the intent to dispatch. Either `action` or `intentId` is required.
  *   - `documentHandle` - The document handle containing document ID, type, and either:
  *     - `projectId` and `dataset` for traditional dataset sources, like `{documentId: '123', documentType: 'book', projectId: 'abc123', dataset: 'production'}`
- *     - `source` for media library or dataset sources, like `{documentId: '123', documentType: 'asset', source: mediaLibrarySource('ml123')}`
+ *     - `source` for media library, canvas, or dataset sources, like `{documentId: '123', documentType: 'asset', source: mediaLibrarySource('ml123')}` or `{documentId: '123', documentType: 'canvas.document', source: canvasSource('canvas123')}`
```

Comment:
Just for consistency, switching the document type names here:

```suggestion
 *     - `source` for media library, canvas, or dataset sources, like `{documentId: '123', documentType: 'sanity.asset', source: mediaLibrarySource('ml123')}` or `{documentId: '123', documentType: 'sanity.canvas.document', source: canvasSource('canvas123')}`
```